external/storm-hbase/src/main/java/org/apache/storm/hbase/bolt/HBaseLookupBolt.java

```diff
@@ -68,7 +69,13 @@ public void execute(Tuple tuple) {
             collector.ack(tuple);
             return;
         }
-        byte[] rowKey = this.mapper.rowKey(tuple);
+        byte[] rowKey = new byte[0];
+        try {
+            rowKey = this.mapper.rowKey(tuple);
+        } catch (ParseException e) {
+            this.collector.reportError(e);
+            this.collector.fail(tuple);
+        }
         Get get = hBaseClient.constructGetRequests(rowKey, projectionCriteria);
```

Comment:
If the `rowKey(tuple)` call throws `ParseException` should the rest of the method be skipped too?  Can it do anything useful without a valid rowkey?